omnigibson/robots/behavior_robot.py

```diff
@@ -139,11 +140,11 @@ def usd_path(self):
     def model_name(self):
         return "BehaviorRobot"
 
-    @property
+    @classproperty
     def n_arms(self):
```

Comment:
Should this still be `self` and not `cls` if its a class property? not sure.

Reply:
fixed!